contributors/design-proposals/apps/controller_history.md

```diff
@@ -42,7 +42,7 @@ and it can be solved by applying a version control system to manifests. Internal
 revision history must only store revisions to the controller's target state that 
 correspond to live Objects and (potentially) a small, configurable number of 
 prior revisions.
-1. History is scale invariant. A revision to a controller is a modification 
+1. History is scaled invariant. A revision to a controller is a modification 
```

Comment:
I believe "scale invariant" is correct, but correct me if I'm wrong.

Reply:
Yes, I believe what was there is correct too